server/sonar-web/src/main/webapp/WEB-INF/app/views/drilldown/issues.html.erb

```diff
@@ -228,7 +228,7 @@
   window.drilldown = {
     metric: null,
     rule: <% if @rule %>'<%= @rule.key -%>'<% else %>null<% end %>,
-    severity: <% if @severity %>'<%= @severity -%>'<% else %>null<% end %>,
+    severity: <% if @severity %>'<%= @escape_javascript(severity) -%>'<% else %>null<% end %>,
```

Comment:
@ustramooner please fix the typo here, should be `<%= escape_javascript(@severity) -%>`